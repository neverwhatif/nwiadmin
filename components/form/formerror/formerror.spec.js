import React from 'react';
import { shallow } from 'enzyme';

import FormError from './index';

describe('FormError', () => {
    it('should render', () => {
        expect(shallow(<FormError />)).toMatchSnapshot();
    });
});
