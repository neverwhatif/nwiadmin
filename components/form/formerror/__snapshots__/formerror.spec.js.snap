// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormError should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FormError />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": undefined,
      "className": "root",
    },
    "ref": null,
    "rendered": null,
    "type": "p",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
        "className": "root",
      },
      "ref": null,
      "rendered": null,
      "type": "p",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
